### DISCUSSION TITLE
Looking forward to a new IK compute function for quardraped

### DISCUSSION THREAD

**Discussion by Huisouan** [isaac-sim/IsaacLab#978] [Q&A]
I wonder if isaac lab has a function that can solve the IK of the quardraped,like pybullet.calculateInverseKinematics2???

**Comment by Dhoeller19** [comment]
Hi @Huisouan,
Unfortunately, we do not support this for Quadrupeds. However, it should be possible to adapt the current inverse kinematics https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab/omni/isaac/lab/controllers/differential_ik.py for the floating base mode.
